iOS/Extensions/UserDefaults/UserDefaults+Theme.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import Foundation
+
+enum Theme: Int {
+  case device
+  case light
+  case dark
+}
+
+@available(iOS 12.0, *)
```

Comment:
`UIUserInterfaceStyle.dark` was already introduced in iOS 12 because of tvOS. In our case, this is not relevant. So this only needs to be available on iOS 13 and later.
In fact, the whole `Theme` type can be restricted to iSO 13 and later, right?